scripts/package-conf-osxpkg.yml

```diff
@@ -0,0 +1,140 @@
+# General
```

Comment:
This config is out of date and should be copied from `conf/config.yml` instead. 
However, this approach of copying the whole file seems like a pain to maintain, (and we've failed to do it for the other packages) so I'd say you should remove all the options that are not the default. Perhaps the packaging file could copy the default config into a neighbour file e.g.
```
cp deepstream/conf/config.yml /opt/.../deepstream/conf/default.yml
```
so that users can see what options can be changed?